opensearch-observability/src/test/kotlin/org/opensearch/observability/bwc/TABackwardCompatibilityIT.kt

```diff
@@ -7,10 +7,9 @@ package org.opensearch.observability.bwc
 
 import org.junit.Assert
 import org.opensearch.common.settings.Settings
-import org.opensearch.observability.ObservabilityPlugin.Companion.BASE_NOTEBOOKS_URI
+import org.opensearch.observability.*
+//import org.opensearch.observability.ObservabilityPlugin.Companion.BASE_NOTEBOOKS_URI
```

Comment:
Do we need comments at here and following lines? Please remove if we don't need.